README.md

```diff
@@ -99,12 +99,13 @@ Usage: swach azure [OPTIONS]
   Cleanup Azure provider
 
 Options:
-  --all    Remove all unused Resources from the provider
-  --nics   Remove only unused NICs from the provider
-  --discs  Remove only unused DISCs from the provider
-  --vms    Remove only unused VMs from the provider
-  --pips   Remove only PiPs from the provider
-  --help   Show this message and exit.
+  --all             Remove all unused Resources from the provider
+  --all_if_older    Remove provider group if all resources are older than SLA
```

Comment:
I am still filling for `all_if_older` :)

Cant, we rename something like `rg`? the description of that option is enough to tell what its going to delete.